Update README.MD for prototype version 7 documentation (#171)
alphagov/forms: design/prototype-version-7/README.MD

# Prototype version 7

Not tested 

## Status

Superseded by [version 8](../prototype-version-8)

___

## Contents

- [Context](#context)
- [Admin interface screenshots](#admin-interface-screenshots)
- [New journeys we introduced](#new-journeys-we-introduced)
- [Form runner screenshots](#form-runner-screenshots)
- [What we learned](#what-we-learned)
- [Opportunites](#opportunities)

___

<br>

## Context

> > In this prototype version, we introduced new user journeys and made content improvements to some pages, based on findings from usability testing in Sprint 9 on prototye version 6. 

Reviewer: ```suggestion
> In this prototype version, we introduced new user journeys and made content improvements to some pages, based on findings from usability testing in Sprint 9 on prototype version 6. 
```

This is the task list page for a live form. This is the beginning of the form creator's journey returning to a live form to edit it. 

<br>

### Add and edit questions (make changes to a live form)

![add and edit questions to make changes to a live form page. Screenshot](screenshots/801-are-you-sure-you-want-to-edit-live-questions.png)
*Page with heading that says “Add and edit questions”.*

There is text saying “Your form is live“ 

Followed with a paragrah descibing what happens when changes are made to a live form “Any changes you make to a live form will be updated in the form immediately.“ 
This is followed by a paragraph describing how any changes will affect people filling the form “This could have an impact on people who are filling in the form at the same time. They may lose any answers they have already provided and may need to start again.“

There is a secondary heading “Are you sure you want to edit the questions?“ followed by two radio options, “Yes” and “No”. 

At the end of the page is a green “Continue” button. 

<br>

### Add and edit questions (on a live form) 

> ![add and edit questions to a live form page. Screenshot](screenshots/802-form-index-live.png)

Reviewer: ```suggestion
![Add and edit questions of a live form page. Screenshot](screenshots/802-form-index-live.png)
```

> *Page with a blue notification banner with title “Important“, followed by “Apply for a juggling licence” caption above the heading that says “Add and edit your questions”.*

Reviewer: ```suggestion
*Page with a blue notification banner titled “Important”, followed by “Apply for a juggling licence” caption above the heading “Add and edit your questions”.*
```

> At the start of the page is a blue notification banner with title “Important“. Inside the notification banner there is a secondary heading saying “Any changes you make to a live form will be updated in the form immediately.“ 

Reviewer: ```suggestion
Inside the notification banner there is a secondary heading saying “Any changes you make to a live form will be updated in the form immediately.“ 
```

This is followed by a paragraph describing how any changes will affect people filling the form “This could have an impact on people who are filling in the form at the same time. They may lose any answers they have already provided and may need to start again.“

> Below the heading is a green status tag “LIVE“ to let the form creator know this is a live form. 

Reviewer: ```suggestion
Below the heading is a green “LIVE“ status tag letting the form creator know this is a ‘live’ form. 
```

There is a green “Add a question” button followed by a secondary heading, “Your questions" and below is a link "Preview the form in a new tab". 

There is a list of questions created and saved for this form. 

> The first question is "What is your name?". To the right is a grey “Move down” button and below it a link "edit".

Reviewer: ```suggestion
The first question is “What is your name?”. To the right is a grey “Move down” button and below it an “Edit” link.   
  
```

> There is a section break

Reviewer: Not sure this is needed?

Author: Yes, have deleted

> The second question is "What is your date of birth?". To the right is a grey “Move up” button, next to it a grey "Move down" button. Bellow the "Mode down" is a link "edit".

Reviewer: ```suggestion
The second question is “What is your date of birth?”. To the right is a grey “Move up” button, next to it a grey “Move down” button. Below the “Move down” button is an “Edit” link.   
  
```

> There is a section break

Reviewer: Not sure this is needed?

> The third queston is "What is your National Insurance number?". To the right is a grey "Move up" button and below is a link "edit".

Reviewer: ```suggestion
The third question is “What is your National Insurance number?”. To the right is a grey “Move up” button and below is an “Edit” link.   
```

<br>
<br>

## Some things we changed since last time

> - Added draft and live status tags to let a user know what state a form is in

Reviewer: ```suggestion
- Added ‘draft’ and ‘live’ status tags to let form creators know what state a form is in  
```

> - Content changes on the example heading and text input label on the edit confirmation / form submitted page

Reviewer: ```suggestion
- Content changes on the ‘example’ heading and textarea label on the “Form submitted page” - confirmation of a submission page  
```

> - Content changes on the example heading and text input label on the edit add a declaration / check your answers page

Reviewer: ```suggestion
- Content changes on the ‘example’ heading and textarea label on the “Add a declaration” - check your answers page
```

> - Updated the second level heading 'Add and edit your questions' page to 'Your questions'

Reviewer: ```suggestion
- Updated the second level heading “Add and edit your questions” page to “Your questions”  
```

> - Content updates on the 'edit question page' 

Reviewer: ```suggestion
- Content updates on the “Edit question” page  
```

> - [Made changes to the Create a form (task list) page](https://github.com/alphagov/forms/tree/documenting-prototype-version-7/design/prototype-version-7/task-list)

Reviewer: ```suggestion
- [Made changes to the “Create a form” task list page](task-list)  
```

- [Introduced new journeys](#new-journeys-we-introduced)

For more information, see [v0.0.7 release notes](https://github.com/alphagov/forms-prototypes/releases/tag/v0.0.7). 

<br>

___

## Form runner screenshots

Below are the screens the form filler (the end user) would see as they complete the form.

<br>

### Preview question 1

![Preview What is your name question page. Screenshot](screenshots/1001-page-preview-new-tab-1.png)

> *Page with “What is your name?” question as a label for a text input. There is a green “Continue” button and a details summary text “Get help with this form“ at the bottom.*

Reviewer: ```suggestion
*Page with “What is your name?” question as the label for a text input. There is a green “Continue” button and a details component with the text “Get help with this form“ at the bottom.*
```

### Preview final question

![Preview What is your National Insurance number question page. Screenshot](screenshots/1002-page-preview-new-tab-3.png)

> *Page with “What is your National Insurance number?” question as a label for a text input, followed below by a green “Continue” button.*

Reviewer: ```suggestion
*Page with “What is your National Insurance number?” question as the label for a text input, followed below by a green “Continue” button.*
```

> There is a details summary link text that says “Get help with this form“ that is expanded to reveal the third level heading “Email“ and an example of a contact email address “support@email.gov.uk“. We added this details component of the contact details that can be used by a person filling a form to get help.

Reviewer: ```suggestion
There is a details component that says “Get help with this form“ that is now expanded. This reveals a third level heading, “Email“, and an example of a contact email address “support@email.gov.uk“. We added this details component for the contact details that can be used by a person filling a form to get help.  
```

> This is the contact details added by the form creator within the admin side of the builder, meaning it can be customised. The form creator can also provide a link to contact support form or a phone number or all three options.

Reviewer: ```suggestion
This is the contact details added by the form creator within the admin side of the builder. The form creator can choose to provide a link to a contact support form or a phone number or any combination of the three contact options.  
```

There is a new visually recessive ‘watermark’ saying “DRAFT” across the background of the page.  

### Preview check your answers (summary page)

![Preview check your answers page. Screenshot](screenshots/1003-check-answers-page-preview-new-tab.png)
*Page with “Check your answers before submitting your form” heading followed by a summary list component.*

> The summary list component lists rows showing the form’s full questions. There is a space to the right where the form filler's answer would appear: this has placeholder text saying “Not completed”. Finally, there is a “Change” link for the form filler to correct or change any answer they feel is incorrect.

Reviewer: ```suggestion
Each row shows the full question text added to the form followed by a space to the right where the form filler’s answer would appear. This currently has placeholder text saying “Not completed”. Finally, there is a “Change” link for the form filler to change any answer they feel is incorrect.  
```

There is a visually recessive ‘watermark’ saying “DRAFT” which stretches across the background of the page. 

### Preview your form has been submitted (confirmation page)

![Preview form submitted page. Screenshot](screenshots/1004-confirmation-page-preview-new-tab.png)
*Page with “Your form has been submitted” heading in a green box.*

This page includes a secondary heading “What happens next” followed by the content “We'll send you an email to let you know the outcome. You'll usually get a response within 10 working days.” This text is editable by the form creator within the admin side of the builder, meaning it can be customised as to the needs of the different forms or department and should match their internal service level agreements (SLAs).

There is a visually recessive ‘watermark’ saying “DRAFT” across the background of the page - this is mostly hidden behind the green success panel.  

<br>
<br>

## Some things we changed since last time
- We added a details component on the question page to reveal the contact details provided by the form creator. The contact details can be used by a person filling a form to get help. 
<br>

___

[Back to the top](#prototype-version-7)

> 
> 
> 
> 
> 
> 
> 
> 
> 
> 
> 
> 
> 
> 
> 
> 
> 
> 
> 
> 
> 
> 
> 
> 
> 

Reviewer: ```suggestion
```
